src/app/xapian/searchservice.ts

```diff
@@ -602,6 +602,9 @@ export class SearchService {
    * @param destinationfolderPath
    */
   moveMessagesToFolder(messageIds: number[], destinationfolderPath: string) {
+    if (!this.api) {
```

Comment:
Does it deserve a warning here maybe? We could also wait for `initSubject()` to make sure we execute it as soon as it's possible.

Reply:
This is the "if not using local index" guard, as used elsewhere in the code (it was just missing here for some reason)